src/content/en/updates/2018/07/pwacompat.md

```diff
@@ -63,18 +63,23 @@ async loaded bundle:
 ```html
 <link rel="manifest" href="manifest.webmanifest" />
 <!-- include PWACompat _after_ your manifest -->
-<script async src="https://cdn.jsdelivr.net/npm/pwacompat@2.0.9/pwacompat.min.js"
-    integrity="sha384-VcI6S+HIsE80FVM1jgbd6WDFhzKYA0PecD/LcIyMQpT4fMJdijBh0I7Iblaacawc"
-    crossorigin="anonymous"></script>
+<script async src="https://unpkg.com/pwacompat" crossorigin="anonymous"></script>
```

Comment:
How about recommending to load it conditionally?

```js
if (/\b(iPad|iPhone|iPod)\b/.test(navigator.userAgent)) {
  import('https://unpkg.com/pwacompat');
}
```

Reply:
I've instead added a "Note: " at the bottom of this section.